### PR TITLE
Replace Objective-c code with Swift syntax.

### PR DESCRIPTION
**Pull request by hardeep-singh**
Replace Objective-c code with Swift syntax. Classes names are Country, CountryCell and CountryManager.

### CODE REVIEW

File: CountryPicker/CountryPicker/CountryManager.swift [hunk 1]
```diff
@@ -69,26 +65,32 @@ open class CountryManager {
...
     func loadCountries() throws {
         
-        guard let countriesFilePath = countriesFilePath,
-              let countryCodes = NSArray(contentsOfFile: countriesFilePath) as? [String] else {
-              throw "Missing array of countries plist in CountryPicker"
+        guard let countriesFilePathIs = countriesFilePath else {
+            return
+        }
+        
+        let url = URL(fileURLWithPath: countriesFilePathIs)
```
Comment (Sharkesm) [inline CountryPicker/CountryPicker/CountryManager.swift]: Keep the same variable name for `countriesFilePath`

**Comment by SURYAKANTSHARMA** [comment]
@hardeep-singh  please run test it failing in ci

**Comment by Sharkesm** [comment]
@hardeep-singh Cross check supported iOS Simulators on Travis

**Comment by SURYAKANTSHARMA** [comment]
@hardeep-singh  Thanks for contribution. `For future` It is showing 7 files changes could, Please don't push pbxproj or unnecessary file to avoid conflicts

**Pull request merged**